src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -31,7 +31,7 @@
 
     @end
 
-    def sample_{@apiMethod.name}({@sampleFuncParams(sample.initCode.argDefaultParams)}):
+    def sample_{@apiMethod.name}({@formalArgs(sample.initCode.argDefaultParams)}):
```

Comment:
We can do this in a separate CL, but I think it would make things clearer to rename `sample.initCode.argDefaultParams`. As it's used here, these are just the sample param definitions, including but not limited to their default values. Maybe the terminal field should be just `.sampleParams`.

Like I said, we can do this separately.